### ISSUE TITLE
train.py的使用疑惑

### ISSUE THREAD

**Issue by wuyangdut** [LongJun123456/Tensorflow-Resnet-Image-Classification#4]
你好，我按照您的教程，发现自己的输出类似这样“step 760 , train loss: 4.231677 , train accuracy: 0.000000, learning_rate: 0.000966”，怎么几乎都不对呢？
